packages/components/pages/f-account-info/src/components/AccountInfo.vue

```diff
@@ -153,12 +162,22 @@ export default {
         };
     },
 
-    async mounted () {
-        await this.initialise();
+    watch: {
+        isAuthFinished () {
```

Comment:
Consider using the immediate: true option to trigger the callback immediately with the current value of the expression: https://vuejs.org/v2/api/#vm-watch